src/cloudinary.js

```diff
@@ -693,6 +693,32 @@ class Cloudinary {
     return this;
   }
 
+  /**
+   * Sets src to null when not using native lazy load
+   * @param img
+   * @param options
+   */
+  handleLazyLoad(img, options){
```

Comment:
@strausr , there's a alignment between the docString and the method name. 
If what the docString says this method is doing is right, then let's rename the method. 
`handleLazyLoad` sounds like it's doing the entire logic

Reply:
This we'll currently be only used on the Angular SDK. When our Lazy load will be implemented on React and Vue it will be used there as well.